### PR TITLE
Add an HTTP server for liveness and readiness probes

### LINKED ISSUE #64: Exposing HTTP endpoints for readiness and liveness probes
In order to check it's liveness and readiness on a Kubernetes environment, the MQTT bridge should have an embedded HTTP server with endpoints for readiness and liveness probes.

### PR DESCRIPTION
**Pull request by ppatierno**
This PR fixes #64
It adds and HTTP server for handling liveness and readiness probes.

### CODE REVIEW

File: packaging/install/020-ConfigMap.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ data:
     mqtt.host=0.0.0.0
     mqtt.port=1883
     #Apache Kafka common
-    kafka.bootstrap.servers=<kafka-bootstrap-servers>
+    kafka.bootstrap.servers=my-cluster-kafka-bootstrap:9092
```
Comment (scholzj) [inline packaging/install/020-ConfigMap.yaml:16]: Was this intentional?

**Review by scholzj — APPROVED** [review]
LGTM apart from the one comment.

File: packaging/install/020-ConfigMap.yaml (hunk 1 above)
Comment (ppatierno) [reply packaging/install/020-ConfigMap.yaml:16]: Yes, it seems we have an "actual" hostname in other components like the canary for example.
We are using the kind of "dafault" from our examples having `my-cluster` as cluster name.

Comment (scholzj) [reply packaging/install/020-ConfigMap.yaml:16]: Ok.

**Review by kyguy — APPROVED** [review]

**Pull request merged**